### PR TITLE
3536: Link up add org button

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/dashboard/template.test.js [hunk 1]
```diff
@@ -22,35 +47,58 @@ describe('organisations dashboard page', () => {
...
+      expect(row1OrgName.text().trim()).toEqual(context.organisations[0][0].data);
+      expect(row1OrgName.attr('href')).toEqual(context.organisations[0][0].href);
+      expect(row1OdsCode.length).toEqual(1);
+      expect(row1OdsCode.text().trim()).toEqual(context.organisations[0][1].data);
+      expect(row2.length).toEqual(1);
+      expect(row2OrgName.length).toEqual(1);
+      expect(row2OrgName.text().trim()).toEqual(context.organisations[1][0].data);
+      expect(row2OrgName.attr('href')).toEqual(context.organisations[1][0].href);
+      expect(row2OdsCode.length).toEqual(1);
+      expect(row2OdsCode.text().trim()).toEqual(context.organisations[1][1].data);
```
Comment (alibombaye) [inline app/pages/dashboard/template.test.js:101]: Over here can we go with hardcoding the expectations instead.

File: app/pages/organisation/template.test.js [hunk 2]
```diff
@@ -159,13 +165,52 @@ describe('organisations dashboard page', () => {
...
+      expect(row1Tag.length).toEqual(0);
+      expect(row2Name.length).toEqual(1);
+      expect(row2Name.text().trim()).toEqual(mockContext.users[1][0].data);
+      expect(row2Name.attr('href')).toEqual(mockContext.users[1][0].href);
+      expect(row2Phone.length).toEqual(1);
+      expect(row2Phone.text().trim()).toEqual(mockContext.users[1][1].data);
+      expect(row2Email.length).toEqual(1);
+      expect(row2Email.text().trim()).toEqual(mockContext.users[1][2].data);
+      expect(row2Tag.length).toEqual(1);
+      expect(row2Tag.text().trim()).toEqual(mockContext.users[1][3].tag.text);
```
Comment (alibombaye) [inline app/pages/organisation/template.test.js:213]: again here if we can hardcode the expectations here.

File: app/pages/dashboard/template.test.js (hunk 1 above)
Comment (kat1906) [reply app/pages/dashboard/template.test.js:101]: how come?

Comment (kat1906) [reply app/pages/dashboard/template.test.js]: I have done it but personally I like to know where the data came from so you can tell if it is hard coded in the component, comes from the context or comes from the manifest

**Pull request merged**